static/_redirects

```diff
@@ -511,7 +511,7 @@
 /concepts/overview/prisma-in-your-stack/graphql /docs/orm/overview/prisma-in-your-stack/graphql
 /concepts/overview/prisma-in-your-stack/rest /docs/orm/overview/prisma-in-your-stack/rest
 /accelerate/concepts /docs/accelerate
-/data-platform/accelerate/concepts /docs/accelerate/what-is-accelerate
+/data-platform/accelerate/concepts /docs/accelerate
```

Comment:
I don't think we should remove existing destinations in our redirects. Open to be wrong though.

Reply:
Ah, good question and I'm not sure about best practices here!

My reasoning is that the `what-is-accelerate` and `what-is-pulse` pages are deleted. So redirecting to these pages doesn't make sense any more IMO. It would lead to a double-redirect and I'm not sure if that's desirable.

Do you have thoughts around this? Is a "double-redirect" ok? In that case, I'm happy to revert this, will trust your judgement here 🙂

Reply:
Well if the pages are deleted than it should be like this 
`/data-platform/accelerate/concepts /docs/accelerate`
and
`/docs/accelerate/what-is-accelerate /docs/accelerate` 

That way the old deleted page gets redirected and the other redirect changes its location. Its only 1 redirect for each

Reply:
> That way the old deleted page gets redirected and the other redirect changes its location. Its only 1 redirect for each

That was my thinking and I believe also what I did in the redirects file, right?

@jharrell do you have any concerns with this?

Reply:
Ah, I understand now. Thank you @nikolasburk, that makes a lot more sense. I think as you did it is correct.